site/css/style.css

```diff
@@ -157,22 +157,29 @@ a {
 }
 
 button,
-input {
+input,
+select {
     background-color: var(--color-bg-secondary);
     color: var(--color-text-secondary);
     border-width: medium;
     cursor: pointer;
 }
 
 button:disabled,
-input:disabled {
+input:disabled,
+select:disabled {
     background-color: var(--color-bg-secondary-low);
     color: var(--color-text-secondary-low);
     cursor: no-drop;
 }
 
 button {
     border-radius: 5px;
+    padding: 0.25em;
+}
+
+select {
+    max-width: 95vw;
```

Comment:
Make this one more specific to your view, we don't want it on all selections.
(This is not because we want a bunch of huge selects, but just do we don't get collisions if we want smaller selects.)